MBBSEmu/Program.cs

```diff
@@ -199,6 +199,9 @@ static void Main(string[] args)
                 var globalCache = _serviceResolver.GetService<IGlobalCache>();
                 var fileHandler = _serviceResolver.GetService<IFileUtility>();
 
+                //Set default manual cleanup flag
+                globalCache.Set("SYSOPGLOBAL-CLEANUP", false);
```

Comment:
If you change the code in MbbsHost to the TryGet<T> you don't need to set an initial value